app/Http/Controllers/EmployeeController.php

```diff
@@ -28,9 +28,11 @@ public function draw(Request $request){
         $draw = $request->query('draw', 1);
         $start = $request->query('start', 0);
         $length = $request->query('length', 10);
-        $totalEmployees =   Employee::count();
-        $arrayColumns = ['','company.name', 'company.branch', 'name', 'position', 'dob', 'email', 'phone', 'address', 'bankAccount.account_no'];
+        $totalEmployees = Employee::count();
+        $arrayColumns = ['', 'company_name', 'branch', 'name', 'position', 'dob', 'email', 'phone', 'address', 'account_no'];
         $arr = array();
+
+        //checking if dropdown filter is filled
         if(is_null($search)){
```

Comment:
make a direct search like `if request has value search` looping over the index method is not recommended.